UI/js-src/lsmb/Reconciliation.js

```diff
@@ -0,0 +1,31 @@
+define("lsmb/Reconciliation",
+       ["dojo/_base/declare",
+        "dojo/topic",
+        "dojo/query",
+        "lsmb/Form",
+        "dijit/_Container",
+        "dojo/NodeList-dom",     // To load extensions in query
+        "dojo/domReady!"],
+       function(declare, Topic, Query, Form, _Container) {
```

Comment:
you use `Topic` and `Query` here, but all other files in our code base us `topic` and `query` (no capitals)